crates/editor_ui/src/change_chain.rs

```diff
@@ -1,14 +1,14 @@
 use bevy::prelude::*;
 
-use super::{editor_tab::EditorTab, EditorUiAppExt};
+use space_editor_tabs::prelude::*;
 use space_undo::ChangeChain;
 
 pub struct ChangeChainViewPlugin;
 
 impl Plugin for ChangeChainViewPlugin {
     fn build(&self, app: &mut App) {
         app.editor_tab_by_trait(
-            super::editor_tab::EditorTabName::Other("Change Chain".to_string()),
+            EditorTabName::Other("Change Chain".to_string()),
```

Comment:
UX suggested that `Change Chain` is a non intuitive name for Undo/Redo logic, Maybe we could call it `Undo Chain`, `Undo/Redo Config` or `Undo Max Chain`?

Reply:
We can! But in another PR, this PR only about code moving